scripts/linters/warranted_angular_security_bypasses.py

```diff
@@ -27,4 +27,6 @@
 #  that the introduction of bypassSecurityTrust was the only option.
 
 EXCLUDED_BYPASS_SECURITY_TRUST_FILES = ()
-EXCLUDED_BYPASS_SECURITY_TRUST_DIRECTORIES = ()
+EXCLUDED_BYPASS_SECURITY_TRUST_DIRECTORIES = (
+    'core/templates/components/forms/custom-forms-directives/thumbnail-display.component.ts', # pylint: disable=line-too-long
```

Comment:
Why are these in the "directories" constant?

Reply:
Changed it.